cfme/web_ui/paginator.py

```diff
@@ -16,9 +16,8 @@
 _page_cell = '//td//td[contains(., " of ")]|//li//span[contains(., " of ")]'
 _check_all = Input("masterToggle")
 
-_prefix = r"(?:Item|Items|Rows|Row)*\s*"
-_regexp = {version.LOWEST: r"{}\((?P<first>\d+)-(?P<last>\d+) of (?P<total>\d+)\)".format(_prefix),
-           "5.5": r"Showing (?P<first>\d+)-(?P<last>\d+) of (?P<total>\d+) (?:item|items?)?"}
+_prefix = r"(?:Item|Items|Rows|Row|Showing)*\s*"
```

Comment:
Actually shouldn't it be rather `?` than `*` as I am looking at it now? btw you can DRY here, `Items?|Rows?|Showing`.